docs/examples/types_secret_types.py

```diff
@@ -3,19 +3,27 @@
 class SimpleModel(BaseModel):
     password: SecretStr
     password_bytes: SecretBytes
+    password_idempotent: SecretStr
```

Comment:
do you need another field to cover the idempotent case?

Surely just a note in the docs should be enough?

Reply:
I added a note in the latest commit and removed this field.